docs/pages/database-access/guides/dynamodb.mdx

```diff
@@ -0,0 +1,308 @@
+---
+title: Database Access with AWS DynamoDB
+description: How to access AWS DynamoDB with Teleport.
+---
+
+<Admonition type="note" title="Important">
```

Comment:
I think it's okay to remove this Admonition. We can work this information into the introductory content.